testing/src/test/java/org/oppia/android/testing/FakeExceptionLoggerTest.kt

```diff
@@ -90,6 +87,7 @@ class FakeExceptionLoggerTest {
 
     assertThat(exception).isInstanceOf(NoSuchElementException::class.java)
   }
+
```

Comment:
Nice find.

Reply:
Yeah, reformatting the file added the missing line here.